Resources/Textures/SS220/Clothing/Neck/Cloaks/Blueshield.rsi/meta.json

```
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "EULA/CLA with a hosting restriction, full text: https://raw.githubusercontent.com/SerbiaStrong-220/space-station-14/master/CLA.txt",
+  "copyright": "Createed based on SS13 Paradise by MrALPAKA for ss220",
```

```suggestion
  "copyright": " Created by gofinodzo (Discord) for SS220",
```
я не понимаю. Текстура бралась и перерисовывалась из сс13? Вообщем, везде, где текстуры полностью свои, укажи лицензию и описание, как тут.
мифологический ник не подойдёт, так как нужна возможность найти любого автора, так что так.